src/Conf.ml

```diff
@@ -1519,17 +1519,18 @@ let parse_line config ~from s =
               , Format.sprintf "expecting %s but got %s" Version.version
                   value ))
     | name, `File x ->
-        if !disable_conf_files then
-          Error (`Malformed "Disabled file configuration of ocamlformat")
-        else
-          C.update ~config
-            ~from:(`Parsed (`File x))
-            ~name ~value ~inline:false
+        C.update ~config
+          ~from:(`Parsed (`File x))
+          ~name ~value ~inline:false
     | name, `Attribute ->
-        if !disable_conf_attrs && not (String.equal name "disable") then
-          Error
-            (`Malformed
-              "Disabled configuration of ocamlformat in attributes")
+        if
+          !disable_conf_attrs
+          && (not (String.equal name "disable"))
```

Comment:
This is very janestreet related, I don't think we should specialize "disable" just yet.